app/controllers/reviews_controller.rb

```diff
@@ -1,63 +1,110 @@
 class ReviewsController < ApplicationController
   before_action :set_review, only: [:show, :edit, :update, :destroy]
 
-  # GET /reviews
-  # GET /reviews.json
+  # GET /restaurants/:restaurant_id/reviews
+  # GET /restaurants/:restaurant_id/reviews.xml
```

Comment:
XML? So hard core 😃 
Why not json?

Reply:
very TRUE!

Misused the copy=paste mechanism.